packages/reference/src/common/EntityStore.tsx

```diff
@@ -188,7 +188,12 @@ const isEntityQueryKey = (queryKey: QueryKey): queryKey is EntityQueryKey => {
   );
 };
 
-type ResourceQueryKey = [ident: 'Resource', resourceType: string, urn: string];
+type ResourceQueryKey = [
+  ident: 'Resource',
+  resourceType: string,
+  urn: string,
+  locale: string | undefined
```

Comment:
Was this done to make sure we always have the locale in the key? Wouldn't it be better to have `locale?: string` ? Or do we need a fixed shape for the key?

Reply:
I think this is required to make the typed array eventually extendable 🤔